libcnb-cargo/tests/test.rs

```diff
@@ -330,5 +329,5 @@ fn copy_dir_recursively(source: &Path, destination: &Path) -> std::io::Result<()
 }
 
 const X86_64_UNKNOWN_LINUX_MUSL: &str = "x86_64-unknown-linux-musl";
-const TARGET_DIR_NAME: &str = "target";
 const CARGO_LIBCNB_BINARY_UNDER_TEST: &str = env!("CARGO_BIN_EXE_cargo-libcnb");
+const DEFAULT_PACKAGE_DIR_NAME: &str = "libcnb-packaged";
```

Comment:
I think it would be worth adding `libcnb-packaged/` to this repo's `.gitignore`.

Reply:
Good call, [268a193](https://github.com/heroku/libcnb.rs/pull/583/commits/268a193f96861684cf23c4af9ebbc1f0d9507fa3)